pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -551,13 +551,18 @@ cdef class {{name}}HashTable(HashTable):
         return_inverse : bool, default False
             Whether the mapping of the original array values to their location
             in the vector of uniques should be returned.
+        use_result_mask: bool, default False
```

Comment:
Nit: Would `return_result_mask` be a clearer name here?

Reply:
Both would work, but the mask isn’t used if it’s false, So I think this is a bit clearer?

Reply:
Gotcha. Yeah keeping as is then makes sense.